x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/osquery/index.tsx

```diff
@@ -48,7 +48,7 @@ const TabComponent = (props: TabProps) => {
 
     return (
       <TabContent>
-        <OsqueryAction metadata={metadata} />
+        <OsqueryAction agentId={metadata?.info?.agent?.id} />
```

Comment:
Just wonder what happens if metadata is not defined, do we show the empty tab content?